configuration/configure-custom-prometheus-metrics.adoc

```diff
@@ -0,0 +1,174 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="configure-custom-prometheus-metrics"]
+= Configuring custom Prometheus metrics
+include::modules/common-attributes.adoc[]
+:context: configure-custom-prometheus-metrics
+
+toc::[]
+
+[role="_abstract"]
+With {product-title}, you can enable some Prometheus metrics to be exposed on the `/metrics` path of the central API endpoint.
+
+The available metric categories include:
+
+* Image vulnerabilities;
+* Node vulnerabilities;
+* Policy violations;
+* Total policy numbers;
+* Cluster health;
+* Certificate expiry.
+
+
+Each category is configured with a data gathering period and a list of metrics. A metric is associated with a set of labels.
+
+[NOTE]
+====
+The `/metrics` path is accessible for API tokens with Administration resource view permissions. The values of the labels are subject to Scoped Access Control.
+====
+
+The currently enabled metrics, if any, are shown on the *System Configuration* page of the *Platform Configuration* section of the {product-title} UI. In edit mode, the UI allows you to enable or disable one of the predefined metrics.
+
+The `/v1/config` API allows for configuring metrics with custom metric names and a custom set of labels.
+
+Accepted image vulnerability labels:
+
+* Cluster
+* Namespace
+* Deployment
+* IsPlatformWorkload
+* ImageID
+* ImageRegistry
+* ImageRemote
+* ImageTag
+* Component
+* ComponentVersion
+* OperatingSystem
+* CVE
+* CVSS
+* Severity
+* EPSSPercentile
+* EPSSProbability
+* IsFixable
+
+Accepted node vulnerability labels:
+
+* Cluster
+* Node
+* Kernel
+* OperatingSystem
+* OSImage
+* Component
+* ComponentVersion
+* CVE
+* CVSS
+* Severity
+* EPSSPercentile
+* EPSSProbability
+* IsFixable
+* IsSnoozed
+
+Accepted policy violation labels:
+
+* Cluster
+* Namespace
+* Resource
+* Deployment
+* IsDeploymentActive
+* IsPlatformComponent
+* Policy
+* Categories
+* Severity
+* Action
+* Message
+* Stage
+* State
+* Entity
+* EntityName
+
+The total policy numbers are exposed as `rox_central_cfg_total_policies` metric, and contains the following label:
+
+* Enabled
+
+The cluster health metric is exposed as `rox_central_health_cluster_info`, and contains the following labels:
+
+* Cluster
+* Type
+* Status
+* Upgradability
+
+The certificate expiry metric is exposed as `rox_central_cert_exp_hours`, and contains the following label:
+
+* Component
+
+== Configuring User Workload monitoring
+
+.Prerequisites
+
+Ensure that the user workload monitoring is enabled in the cluster by checking the content of the `cluster-monitoring-config` `ConfigMap` in the `openshift-monitoring` namespace.
+If the `ConfigMap` does not exist, create it with the following content:
```

Comment:
```suggestion
If the config map does not exist, create it with the following content:
```
I don't think we are allowed to have [procedures in the prerequisite](https://github.com/redhat-documentation/modular-docs/blob/main/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc) so we might have to make this into a separate step under the procedure and mark it (Optional):

For example:

.Prerequisites

* User workload monitoring is enabled in the cluster. Check that the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace includes `enableUserWorkload: true` for `config.yaml`.

Then, as step 1 in procedure:

1. (Optional) Create a `cluster-monitoring-config` config map in the `openshift-monitoring` namespace with the following content:

(yaml)